integration_tests/test_login.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize("username", (
+    "test",
+    "test2"
+))
+async def test_user_existence(test_client, username):
+    """Verify that these users exist on the test server"""
+    client, welcome_message = await test_client(username, "foo")
+
+    assert welcome_message["login"] == welcome_message["me"]["login"] == username
+    assert welcome_message["id"] == welcome_message["me"]["id"]
```

Comment:
That seems a bit of a weird assert but ok.

Reply:
It's apparently legacy behaviour. I'm assuming the python client uses it.